### ISSUE TITLE
Add beginners tools

### ISSUE THREAD

**Issue by MohamedAlFahim** [keras-team/keras#9203]
**Proposed label:** Feature
Beginners should be able to use a simple module within Keras for basic tasks like so:
```python
# Imports a simple feed-forward network model
from keras.simpletools import Simple_feed_forward

# The model has 2 inputs, 3 hidden nodes, and 1 output
model = Simple_feed_forward([2,3,1], activation="tanh")

# Learns XOR
model.fit([[1,1],[0]],
          [[0,1],[1]],
          [[1,0],[1]],
          [[0,0],[0]], 
          optimizer="SGD", iterations=200, show_error=False)

# Predict
print(model.predict([1,1]))
```

**Comment by fchollet** [comment]
Keras has extensive getting-started documentation and education material available.
